zebra-chain/Cargo.toml

```diff
@@ -29,7 +29,7 @@ fpe = "0.5.1"
 futures = "0.3.21"
 group = "0.11.0"
 halo2 = { package = "halo2_proofs", version = "=0.1.0-beta.4" }
-hex = "0.4.3"
+hex = { version = "0.4.3", features = ["serde"] }
```

Comment:
Build time: Why is this dependency required for all builds now?

Reply:
This is a bit orthogonal to the rest of the PR, it's because I ran into a build failure on `main` when doing

```
cargo clean
cargo build -p zebra-utils
```

Reply:
Oh wow yeah that seems like another instance of #1364.

Thanks for the fix!